model/clusters_mgmt/v1/machine_type_type.model

```diff
@@ -19,6 +19,10 @@ class MachineType {
 	// Human friendly identifier of the machine type, for example `r5.xlarge - Memory Optimized`.
 	Name String
 
+	// The category which the machine type is suitable for.
+	// Options: `general_purpose`, `memory_optimized`, `compute_optimized`.
+	Category String
```

Comment:
If it only accepts 3 values, why is it a string and not a type, like we do for state?

Reply:
Make sense. PreTAL.